.github/workflows/go.yml

```diff
@@ -8,11 +8,11 @@ jobs:
     strategy:
       matrix:
         os: [macOS-latest, ubuntu-latest]
-        goversion: [1.13, 1.14, 1.15]
+        goversion: [1.16, 1.17, 1.18]
```

Comment:
Hmm, we should probably make this 1.17, 1.18 and 1.19 if we're going for the last 3 versions.